### PR TITLE
adding batch gets to docs

### PR DESCRIPTION
**Pull request by manas-vessel**

### CODE REVIEW

File: docs/pages/beta/crm-beta/crm-beta.yaml [hunk 1]
```diff
@@ -63,6 +63,65 @@ paths:
...
+                  $ref: "#/components/schemas/IncludeFields"
+                ids:
+                  type: array
+                  items:
+                    type: string
+                cursor:
+                  type: string
+            examples:
+              example-1:
+                value:
```
Comment (zkirby) [inline docs/pages/beta/crm-beta/crm-beta.yaml:118]: this should include the "ids" in here too

File: docs/pages/beta/crm-beta/crm-beta.yaml [hunk 2]
```diff
@@ -63,6 +63,65 @@ paths:
...
+                  include:
+                    $native: false
+                  cursor: "123123"
+      operationId: batch-ready-users
```
Comment (zkirby) [inline docs/pages/beta/crm-beta/crm-beta.yaml:122]:
```suggestion
      operationId: batch-read-users
```

File: docs/pages/beta/crm-beta/crm-beta.yaml [hunk 3]
```diff
@@ -192,6 +251,79 @@ paths:
...
+                cursor:
+                  type: string
+                ids:
+                  type: array
+                  items:
+                    type: string
+            examples:
+              example-1:
+                value:
+                  cursor: "123123"
```
Comment (zkirby) [inline docs/pages/beta/crm-beta/crm-beta.yaml:319]: same for here and all the rest to include "ids"

**Review by zkirby — COMMENTED** [review]
Batch reads are supported for both sf and hs right?

**Comment by manas-vessel** [comment]
> 

Yep, they're supported for both SF and HS. They aren't supported for Lists though, since for both cases they're not very straightforward to batch read

**Review by zkirby — APPROVED** [review]

**Pull request merged**